### DISCUSSION TITLE
Page names are missing in page files

### DISCUSSION THREAD

**Discussion by Cherie-R** [shoonia/velo-filesystem#36] [General]
Hi @shoonia. I'm still using your extension, now for Blocks too, thank-you. :-) I recently noticed the page name is missing from the files in the pages directory. For example, it used to be named "My Page.js" and now it has the id only "qewrt.js". This seems to be regardless of whether or not I have "Include Page ID" checked. Is this something that changed in Wix or can you fix it? :-)
Thank-you, Cherie

**Comment by shoonia** [comment]
Hey, @Cherie-R !
Fixed in v0.9.0

**Reply by Cherie-R** [reply]
Thank-you so much @shoonia!!!!!
